setup.cfg

```diff
@@ -19,6 +19,7 @@ include_trailing_comma=True
 force_grid_wrap=0
 combine_as_imports=True
 line_length=88
+profile=black
```

Comment:
I think this addition here means we don't have to specify `--profile=black` in other placed where `isort` is run. Could you confirm whether or not this is the case and, if so, remove `--profile=black` accordingly?

Reply:
Yes, thanks, shall do! I was on the fence about this, but now that you've brought it up as well, it's probably worth confirming at this stage whether or not it can be picked up properly, and then the redundancies will be eliminated, if that ends up being the case.